### DISCUSSION TITLE
[Public roadmap] Anyone finds use for a button for commits on the repository page?

### DISCUSSION THREAD

**Discussion by MithunKinarullathil** [community/community#11236] [Other Feature Feedback, Questions, & Ideas] [answered]
![Untitled](https://user-images.githubusercontent.com/51120171/153202218-5cc9d117-839d-4497-9e23-099cd8bc1765.png)

Eg: https://github.com/cra-ros-pkg/robot_localization/commits

**Comment by letsfindaway** [comment, marked as answer]
The link is already there, but I also look for it each time: It is the "44 commits" below the green "Code" button which links to the commit history. Is it this you want to have more prominent on the top?

**Reply by MithunKinarullathil** [reply]
Ohh I didn't see that. Thank you. I feel like it's better to be more exposed.
